### DISCUSSION TITLE
Hello Mdanalysis, I am Tanish Yelgoe

### DISCUSSION THREAD

**Discussion by tanishy7777** [MDAnalysis/mdanalysis#4811] [Introductions]
Hello everyone
 👋
I am Tanish Yelgoe, a sophomore pursuing a Bachelor's in Artificial Intelligence at the Indian Institute of Technology, Gandhinagar. I am thrilled to embark on my open-source journey as part of this community. I look forward to learning from fellow developers and contributing meaningfully to this project.

**Comment by RMeli** [comment]
Hi @tanishy7777, welcome to the MDAnalysis community!

We look forward to your contributions. Please have a look at our [User Guide](https://userguide.mdanalysis.org/stable/index.html), and specifically at the [Contributing to MDAnalysis](https://userguide.mdanalysis.org/stable/contributing.html) section, to learn how to best contribute to the project.
